x/campaign/keeper/msg_server_update_total_supply_test.go

```diff
@@ -99,6 +99,15 @@ func TestMsgUpdateTotalSupply(t *testing.T) {
 			},
 			err: types.ErrMainnetInitialized,
 		},
+		{
```

Comment:
Same comment as above

Maybe it can be worth for check all places where `TotalSupply` is assigned for test purposes and use `sample.TotalSupply()`